_topics/en/quantum-resistance.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Quantum resistance
+
+## Optional.  Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+# shortname: foo
+
+## Optional.  An entry will be added to the topics index for each alias
+aliases:
+  - Post-quantum cryptography
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Security Enhancements
+
+## Optional.  Produces a Markdown link with either "[title][]" or
+## "[title](link)"
+#primary_sources:
+#    - title: Test
+#    - title: Example
+#      link: https://example.com
+
+## Optional.  Each entry requires "title", "url", and "date".  May also use "feature:
+## true" to bold entry
+optech_mentions:
+  - title: Discussion of quantum computer attacks on taproot
+    url: /en/newsletters/2021/03/24/#discussion-of-quantum-computer-attacks-on-taproot
+    feature: true
+
+  - title: "Question about paying public keys directly versus hash indirection"
```

Comment:
The other questions were "Question:" prefixed so this one stood out to me.

Reply:
others have been changed to "Question about ..."